{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright Â© Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An update for python-django is now available for Red Hat OpenStack Platform 8.0 (Liberty).\n\nRed Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "Django is a high-level Python Web framework that encourages rapid development and a clean, pragmatic design. It focuses on automating as much as possible and adhering to the DRY (Don't Repeat Yourself) principle.\n\nSecurity Fix(es):\n\n* A redirect flaw, where the is_safe_url() function did not correctly sanitize numeric-URL user input, was found in python-django. A remote attacker could exploit this flaw to perform XSS attacks against the OpenStack dashboard. (CVE-2017-7233)\n\nRed Hat would like to thank the Django project for reporting this issue.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat products and services.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHSA-2017:1470",
				"url": "https://access.redhat.com/errata/RHSA-2017:1470"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/security/updates/classification/#moderate",
				"url": "https://access.redhat.com/security/updates/classification/#moderate"
			},
			{
				"category": "external",
				"summary": "1437234",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1437234"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2017/rhsa-2017_1470.json"
			}
		],
		"title": "Red Hat Security Advisory: python-django security update",
		"tracking": {
			"current_release_date": "2024-11-14T23:34:25+00:00",
			"generator": {
				"date": "2024-11-14T23:34:25+00:00",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "4.2.1"
				}
			},
			"id": "RHSA-2017:1470",
			"initial_release_date": "2017-06-14T15:28:05+00:00",
			"revision_history": [
				{
					"date": "2017-06-14T15:28:05+00:00",
					"number": "1",
					"summary": "Initial version"
				},
				{
					"date": "2017-06-14T15:28:05+00:00",
					"number": "2",
					"summary": "Last updated version"
				},
				{
					"date": "2024-11-14T23:34:25+00:00",
					"number": "3",
					"summary": "Last generated version"
				}
			],
			"status": "final",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "Red Hat OpenStack Platform 8.0 (Liberty)",
								"product": {
									"name": "Red Hat OpenStack Platform 8.0 (Liberty)",
									"product_id": "7Server-RH7-RHOS-8.0",
									"product_identification_helper": {
										"cpe": "cpe:/a:redhat:openstack:8::el7"
									}
								}
							}
						],
						"category": "product_family",
						"name": "Red Hat OpenStack Platform"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-django-0:1.8.18-1.el7ost.src",
								"product": {
									"name": "python-django-0:1.8.18-1.el7ost.src",
									"product_id": "python-django-0:1.8.18-1.el7ost.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django@1.8.18-1.el7ost?arch=src"
									}
								}
							}
						],
						"category": "architecture",
						"name": "src"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-django-0:1.8.18-1.el7ost.noarch",
								"product": {
									"name": "python-django-0:1.8.18-1.el7ost.noarch",
									"product_id": "python-django-0:1.8.18-1.el7ost.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django@1.8.18-1.el7ost?arch=noarch"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-django-bash-completion-0:1.8.18-1.el7ost.noarch",
								"product": {
									"name": "python-django-bash-completion-0:1.8.18-1.el7ost.noarch",
									"product_id": "python-django-bash-completion-0:1.8.18-1.el7ost.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django-bash-completion@1.8.18-1.el7ost?arch=noarch"
									}
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django-0:1.8.18-1.el7ost.noarch as a component of Red Hat OpenStack Platform 8.0 (Liberty)",
					"product_id": "7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.noarch"
				},
				"product_reference": "python-django-0:1.8.18-1.el7ost.noarch",
				"relates_to_product_reference": "7Server-RH7-RHOS-8.0"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django-0:1.8.18-1.el7ost.src as a component of Red Hat OpenStack Platform 8.0 (Liberty)",
					"product_id": "7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.src"
				},
				"product_reference": "python-django-0:1.8.18-1.el7ost.src",
				"relates_to_product_reference": "7Server-RH7-RHOS-8.0"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django-bash-completion-0:1.8.18-1.el7ost.noarch as a component of Red Hat OpenStack Platform 8.0 (Liberty)",
					"product_id": "7Server-RH7-RHOS-8.0:python-django-bash-completion-0:1.8.18-1.el7ost.noarch"
				},
				"product_reference": "python-django-bash-completion-0:1.8.18-1.el7ost.noarch",
				"relates_to_product_reference": "7Server-RH7-RHOS-8.0"
			}
		]
	},
	"vulnerabilities": [
		{
			"acknowledgments": [
				{
					"names": [
						"the Django project"
					]
				}
			],
			"cve": "CVE-2017-7233",
			"cwe": {
				"id": "CWE-79",
				"name": "Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')"
			},
			"discovery_date": "2017-03-29T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "1437234"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A redirect flaw, where the is_safe_url() function did not correctly sanitize numeric-URL user input, was found in python-django. A remote attacker could exploit this flaw to perform XSS attacks against the OpenStack dashboard.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "python-django: Open redirect and possible XSS attack via user-supplied numeric redirect URLs",
					"title": "Vulnerability summary"
				},
				{
					"category": "other",
					"text": "This issue affects the versions of python-django as shipped with Red Hat Satellite 6. Please note that python-django, as used by Pulp does not make use of the Pulp doesn't use \"is_safe_url\" directly or the \"i18n\" views or the \"django.contrib.auth\" Login view. Red Hat Product Security has rated this issue as having Low security impact. A future update may address this issue. For additional information, refer to the Issue Severity Classification: https://access.redhat.com/security/updates/classification/.",
					"title": "Statement"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.noarch",
					"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.src",
					"7Server-RH7-RHOS-8.0:python-django-bash-completion-0:1.8.18-1.el7ost.noarch"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2017-7233"
				},
				{
					"category": "external",
					"summary": "RHBZ#1437234",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1437234"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2017-7233",
					"url": "https://www.cve.org/CVERecord?id=CVE-2017-7233"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2017-7233",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2017-7233"
				}
			],
			"release_date": "2017-04-04T00:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "For details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.noarch",
						"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.src",
						"7Server-RH7-RHOS-8.0:python-django-bash-completion-0:1.8.18-1.el7ost.noarch"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHSA-2017:1470"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "NONE",
						"baseScore": 6.1,
						"baseSeverity": "MEDIUM",
						"confidentialityImpact": "LOW",
						"integrityImpact": "LOW",
						"privilegesRequired": "NONE",
						"scope": "CHANGED",
						"userInteraction": "REQUIRED",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
						"version": "3.0"
					},
					"products": [
						"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.noarch",
						"7Server-RH7-RHOS-8.0:python-django-0:1.8.18-1.el7ost.src",
						"7Server-RH7-RHOS-8.0:python-django-bash-completion-0:1.8.18-1.el7ost.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Moderate"
				}
			],
			"title": "python-django: Open redirect and possible XSS attack via user-supplied numeric redirect URLs"
		}
	]
}
